tests/PHPStan/Rules/Methods/CallStaticMethodsRuleTest.php

```diff
@@ -250,7 +250,7 @@ public function testCallStaticMethods(): void
 				'Learn more at https://phpstan.org/user-guide/discovering-symbols',
 			],
 			[
-				'Call to an undefined static method static(CallStaticMethods\CallWithStatic)::nonexistent().',
```

Comment:
I tried to minimize breaking the previous error messages, and this is the result that I ended up with. IMO `static(...)` is unnecessary information here - the `nonexistent` method has to exist on `CallWithStatic` for the issue to go away.

Should I limit this change to bleeding edge only (e.g. via `newRuleLevelHelper`)?

Reply:
I think it's fine as it is 😊